src/plugins/custom_integrations/public/components/replacement_card/replacement_card.component.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+/** @jsx jsx */
+
+import { css, jsx } from '@emotion/react';
+
+import {
+  htmlIdGenerator,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiText,
+  EuiAccordion,
+  EuiLink,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { CustomIntegration } from '../../../common';
+import { usePlatformService } from '../../services';
+
+export interface Props {
+  replacements: Array<Pick<CustomIntegration, 'id' | 'uiInternalPath' | 'title'>>;
+}
+
+const URL_COMPARISON = 'https://ela.st/beats-agent-comparison';
+const idGenerator = htmlIdGenerator('replacementCard');
+const alsoAvailable = i18n.translate('customIntegrations.components.replacementAccordionLabel', {
+  defaultMessage: 'Also available in Beats',
+});
+
+const link = (
+  <EuiLink href={URL_COMPARISON} data-test-subj="customIntegrationsBeatsAgentComparisonLink">
```

Comment:
nit: I _think_ for outside links, we want to open in a new tab. E.g. the links on the footer that says `Request new integrations using our discuss forum(opens in a new tab or window).`

It shows that little arrow icon.